lib/rbac/filterer.rb

```diff
@@ -262,6 +262,8 @@ def search(options = {})
         if targets.first.kind_of?(Numeric)
           target_ids = targets
           # assume klass is passed in
+        elsif targets.first.kind_of?(Hash)
+          target_ids = targets.first[:objectIds]
```

Comment:
is there a need for a hash here?
can we just use the ids as an array in the url?